packages/terra-dropdown-button/src/_DropdownList.jsx

```diff
@@ -185,7 +190,7 @@ class DropdownList extends React.Component {
         onFocus={this.handleFocus}
         onKeyDown={this.handleKeyDown}
         onKeyUp={this.handleKeyUp}
-        role="menu"
+        role="menubar"
```

Comment:
```suggestion
        role="menu"
```
I think we should fallback to menu !! Apologies for creating the confusion Menubar is more suitable when it consists navigation / link items . dropdown button will always have dropdown.item as child which is more of action items instead of link or navigation items.
https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/

Reply:
Reverted as suggested.